### ISSUE TITLE
Is there a way to specify independent destination paths for hls and dash manifests?

### ISSUE THREAD

**Issue by eck1001** [gpac/gpac#2683]
Hello I'm trying to run dasher in dual mode and specify both hls and dash's manifest path explicitly (different manifest destination directory). Similar to the following:
<img width="148" alt="image" src="https://github.com/gpac/gpac/assets/29557802/828f7232-fcd4-4c8e-ad64-54792801b0fd">

Reading the documentation here, this seems possible: https://github.com/gpac/gpac/wiki/dasher#output-redirecting. Specifically the following verbiage:

"This will create a single dasher whose outputs (manifests and segments) will be redirected to the given URLs.
When explicitly loading the filter, the [dual](https://github.com/gpac/gpac/wiki/dasher#dual) option will be disabled unless [mname](https://github.com/gpac/gpac/wiki/dasher#mname) is set to the alternate output name."

Running the gpac command: 
```gpac  -i someFile.mp4 dasher:dual:mname=/out/dash/dash_live.mpd --segdur=10 --cdur=2 --asto=8 --dmode=dynamic --llhls=br --profile=live --tsb=60 --template='../common/$File$_$Number$' -o /out/hls/hls_live.m3u8 -o /out/dash/dash_live.mpd```

Results in both fout destinations containing a dash manifest (the fout names/paths are correct but the content is dash for both).

Here is the resulting filter graph (paths are simplified above):
```
Filters connected:
fin (src=/Users/ekram/Desktop/ott_assets/Sintel/sintel.mp4) (idx=1)
-(PID sintel.mp4) mp4dmx (dyn_idx=5)
--(PID V1)\
--(PID A2)-> dasher (dual:mname=/Users/ekram/Desktop/test-gpac-multi-out/dash/dash_live.mpd) (idx=2)
---(PID manifest_mpd) fout (dst=/Users/ekram/Desktop/test-gpac-multi-out/hls/hls_live.m3u8:mname=/Users/ekram/Desktop/test-gpac-multi-out/hls/hls_live.m3u8) (idx=3)
---(PID manifest_mpd) fout (dst=/Users/ekram/Desktop/test-gpac-multi-out/dash/dash_live.mpd) (idx=4)
---(PID V1)           mp4mx (dyn_idx=8)
----(PID V1) fout (dst=/Users/ekram/Desktop/test-gpac-multi-out/common/sintel_init_track1_.mp4:gfopt:frag:xps_inband=no:psshs=moov:mime=video/mp4) (idx=6)
---(PID A2)           mp4mx (dyn_idx=9)
----(PID A2) fout (dst=/Users/ekram/Desktop/test-gpac-multi-out/common/sintel_init_track2_.mp4:gfopt:frag:xps_inband=no:psshs=moov:mime=audio/mp4) (idx=7)
```
Note the `PID manifest_mpd` for both dasher manifest outputs.

I have tried multiple variants of the above gpac command in hopes of  getting a working solution. Switching the `fout` / `output` 
around (2nd output is m3u8 and mname is m3u8 file path) results in: 
<img width="218" alt="image" src="https://github.com/gpac/gpac/assets/29557802/00fa80c0-3e6e-4a48-93df-185aa96d4c9c">

Where the `dash_live.mpd` file has cloned contents of the hls manifest file and the sub-playlist hls files are relative to the first fout destination path (dash location, not the hls location). 

(cmd for reference):
```
gpac -graph -i /Users/ekram/Desktop/ott_assets/Sintel/sintel.mp4 dasher:dual:mname=/Users/ekram/Desktop/test-gpac-multi-out/hls/hls_live.m3u8 --segdur=10 --cdur=2 --asto=8 --dmode=dynamic --llhls=br --profile=live --tsb=60 --template='../common/$File$_$Number$' -o /Users/ekram/Desktop/test-gpac-multi-out/dash/dash_live.mpd -o /Users/ekram/Desktop/test-gpac-multi-out/hls/hls_live.m3u8
```

I know running two separate dashers will solve the destination paths detailed above -- but my understanding is that the resulting segment duplication will be a huge performance cost and that is not worth trying to achieve the desired file output structure.

TLDR: Is there a way to specify independent destination paths for hls and dash manifests (and bonus -- dash and top master hls manifest names) when running dasher in `dual` mode?
